openpaas-james/integration-tests/jmap/jmap-integration-tests-common/src/main/scala/com/linagora/openpaas/james/common/probe/JmapGuiceCustomProbe.java

```diff
@@ -17,8 +20,8 @@ public JmapGuiceCustomProbe(FilteringManagement filteringManagement) {
         this.filteringManagement = filteringManagement;
     }
 
-    public void setRulesForUser(Username username, Rule... rules) {
-        Mono.from(filteringManagement.defineRulesForUser(username, rules)).block();
+    public void setRulesForUser(Username username, Optional<Version> ifInState, Rule... rules) {
```

Comment:
We don't need the ifInState for the probe, we can always pass Optional.empty IMO